src/k-provider/ovp/play-source-url-builder.js

```diff
@@ -22,16 +22,16 @@ export default class PlaySourceUrlBuilder {
    */
   static build(urlParams: urlParamsType): string {
     const config = OVPConfiguration.get();
-    const cdnUrl: string = config.cdnUrl;
+    const serviceUrlOrigin: string = new URL(config.serviceUrl).origin;
```

Comment:
seems `URL` is unsupported in IE11
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Thanks, good thing you caught it. Fixed